### PR TITLE
Implement License Manager v2

### PR DESCRIPTION
**Pull request by moorscode**
Uniform plugin name
Use v2 when available fallback to v1

This also fixes the assumption that the License Manager is available on activation or initialisation.
Which isn't when Yoast SEO or Yoast SEO Premium are not activated.

### CODE REVIEW

**Comment by rensw90** [comment]
CR-Done 👍

File: classes/class-wpseo-news.php [hunk 1]
```diff
@@ -248,7 +252,8 @@ public function edit_post_css() {
 	 * @since 2.0.0
 	 */
 	public function error_missing_wpseo() {
-		echo '<div class="error"><p>' . sprintf( __( 'Please %sinstall &amp; activate Yoast SEO%s and then enable its XML sitemap functionality to allow the Yoast News SEO module to work.', 'wordpress-seo-news' ), '<a href="' . esc_url( admin_url( 'plugin-install.php?tab=search&type=term&s=wordpress+seo&plugin-search-input=Search+Plugins' ) ) . '">', '</a>' ) . '</p></div>';
+		/* translators: %1$s resolves to the link to search for Yoast SEO, %2$s resolves to the closing tag for this link */
+		echo '<div class="error"><p>' . sprintf( __( 'Please %sinstall &amp; activate Yoast SEO%s and then enable its XML sitemap functionality to allow the News SEO module to work.', 'wordpress-seo-news' ), '<a href="' . esc_url( admin_url( 'plugin-install.php?tab=search&type=term&s=yoast+seo&plugin-search-input=Search+Plugins' ) ) . '">', '</a>' ) . '</p></div>';
```
Comment (andizer) [inline classes/class-wpseo-news.php]: Please move Yoast SEO outside the language string

File: classes/class-wpseo-news.php [hunk 2]
```diff
@@ -266,7 +271,7 @@ public function error_upgrade_wp() {
 	 * @since 2.0.0
 	 */
 	public function error_upgrade_wpseo() {
-		echo '<div class="error"><p>' . __( 'Please upgrade the Yoast SEO plugin to the latest version to allow the Yoast News SEO module to work.', 'wordpress-seo-news' ) . '</p></div>';
+		echo '<div class="error"><p>' . __( 'Please upgrade the Yoast SEO plugin to the latest version to allow the News SEO module to work.', 'wordpress-seo-news' ) . '</p></div>';
```
Comment (andizer) [inline classes/class-wpseo-news.php]: Please move Yoast SEO outside the language string

**Comment by andizer** [comment]
CR Done 👷

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**